Source/SelfService/Web/microservice/microserviceDetails/configurationFilesSection/setupSection/containerImageFields.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React from 'react';
+
+import { Box, SxProps, Typography } from '@mui/material';
+
+import { Input } from '@dolittle/design-system';
+
+import { HeadArguments } from '../../../components/form/headArguments';
+
+type ContainerImageFieldsProps = {
+    disabled: boolean;
+    sx: SxProps;
+};
+
+export const ContainerImageFields = ({ disabled, sx }: ContainerImageFieldsProps) =>
+    <Box sx={sx}>
+        <Typography variant='subtitle2' sx={{ mb: 2 }}>Container Image Settings</Typography>
+
+        <Input
+            id='headImage'
+            label='Image Name'
+            required
+            disabled={disabled}
+            sx={{ width: 500 }}
```

Comment:
I don't think this will scale very well on smaller screens. I haven't looked at the other inputs, but would it work by using max-width instead so that on smaller screens it continues to shrink?

This is NOT a priority, and we could revisit the mobile experience later.

Reply:
It is not scaling well, indeed :D Bigger issue here is 'Tooltip' if field is focused - 'Tooltip' also take fixed width. Need to come back to it imo :)